src/components/IssueFieldReferences.vue

```diff
@@ -45,19 +45,20 @@ function handleDelete(uuid: string, closeModal: () => void) {
       @item:delete="emit('reference:delete', $event)"
       @item:new="emit('reference:new')"
     >
-      <template #default="{ item: { url, type, updated_dt, ...item } }">
-        <a :href="url" target="_blank">
-          <span class="me-2">{{ updated_dt }}</span>
+      <template #default="{ item: { url, type, ...item } }">
+        <div class="info-group">
           <span
             class="badge rounded-pill"
             :class="{
               'bg-primary': type === 'ARTICLE',
               'bg-warning': type !== 'ARTICLE',
             }"
+            :style="type !== 'ARTICLE' ? 'color: black' : ''"
```

Comment:
I think this is fine for the time being but I'm wondering if we want that black on yellow contrast everywhere--what do you think? We can also lighten the yellow too

Reply:
I think the other elements with similar style are already with black font over yellow bg.
For example the workflow buttons on status field:

![image](https://github.com/RedHatProductSecurity/osim/assets/29104825/8de71f31-485a-42d7-9381-ebf5b2200b28)

No preference from me, we can discuss the style on this "warning like" elements in the future.